packages/renderer/src/lib/pod/DeployPodToKube.svelte

```diff
@@ -80,6 +80,10 @@ async function updatePod() {
   if (createdPod.status?.phase === 'Running') {
     clearInterval(updatePodInterval);
     deployFinished = true;
+    window.telemetryTrack('deployToKube.running', {
```

Comment:
following the other discussion, should you remove `running` from here?

Reply:
I'm not really able to judge it as i never used telemetry extensively to make analysis. Maybe who create graphs, see them daily should tell what are their preferences.

I have just commented based on the other discussion. I tried it and it works for me so it has my approval.

I wonder how this info can be useful though as, if you don't see it in the logs, you don't really know if it was because the user left the page or the pod never run. Isn't it enough to just track the deploy?